packages/components/src/Text/Text.mdx

```diff
@@ -69,3 +69,20 @@ only be used right after a page title.
     Drag to rearrange the order that fields show up in Jobber
   </Text>
 </Playground>
+
+## number Of Lines
+
+Used to truncate large text with `...` based on line wrapping, such that the
+total number of lines does not exceed this number.
+
+<Playground>
+  <Text numberOfLines="3">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum nec pulvinar nunc. Suspendisse nec eros pretium, rutrum purus sit
+    amet, finibus ante. Donec pretium condimentum scelerisque. Duis eu ligula nec metus suscipit feugiat.
+
+    Etiam sapien sapien, mattis eu tincidunt quis, pretium sed metus. Maecenas quis dolor lacinia libero rhoncus fringilla. Cras mi ante,
```

Comment:
This makes it appear as though we can have paragraphs within the `Text` component, which I don't think is the case. Can we remove the empty line?

Reply:
👍  I will remove the paragraph. But we are actually planning to use the Text component to show the notes preview text in activity feed. This preview text is supposed to preserve the paragraph and space formatting that was entered by SPs. Should the Text component not be used in that case?

Reply:
@eddysims

Reply:
@Pavithrak I would think that if the text needs to preserve the paragraph structure that we would want to use `Content` and then wrap each paragraph in `Content`

```ts
<Content>
   <Text>
      ...
   </Text>
   <Text>
      ...
   </Text>
</Content>
```

Or alternatively the `Markdown` component will preserve paragraphs as well.